hooks/pre_prompt.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+
+if __name__ == '__main__':
+    print("\n\n\n")
+    print("#" * 80)
+    print("The napari cookiecutter plugin template has been replaced with a copier template!")
+    print()
+    print("To create a plugin using the new template, run:")
+    print()
+    print("python -m pip install copier jinja2-time")
+    print("python -m pip install npe2")
+    print("copier copy --trust https://github.com/napari/napari-plugin-template new-plugin-name")
+    print()
+    print("For more information, refer to the template README:")
```

Comment:
Sweet!

This works well locally. It's fine to merge as is. 

I think that it makes sense to give the location of the new template's repo before the how to use the new template.

It won't hurt to have the new repo listed twice. Once at the beginning and once at the end.